src/routers/secure/scrape.py

```
@@ -729,6 +759,18 @@ class AutoScrapeRequestPayload(BaseModel):
     ] = None
+    disable_bitrate_check: Annotated[
+        bool,
+        Field(
+            default=False,
+            description="Disable bitrate check for this scrape",
+        ),
+    ] = False
+
+
+async def auto_scrape_item(
+    body: Annotated[AutoScrapeRequestPayload, Body()],
```

Ruff (0.14.10)

Unused function argument: `body`

(ARG001)